src/viam/sdk/components/private/gripper_server.cpp

```diff
@@ -0,0 +1,68 @@
+#include <viam/sdk/components/private/gripper_server.hpp>
+
+#include "gripper_server.hpp"
```

Comment:
I think probably we should drop this `include` line?

Reply:
ugh yes these keep getting put in automatically somehow not sure which tool is doing it 😠